packages/@storefront/breadcrumbs/src/refinement-crumbs/index.ts

```diff
@@ -41,8 +45,8 @@ class RefinementCrumbs {
   updateRefinements = () => this.set(this.selectRefinements(this.state.navigationSelector));
 
   selectRefinements(getNavigation: RefinementCrumbs.NavigationSelector) {
-    const { field } = this.props;
-    const navigation = getNavigation(field);
+    const { field } =  this.props.selectedNavigation || this.props;
```

Comment:
maybe add a comment above both of these to remove the `||` when we deprecate the other stuff?

Reply:
Will do.